README.md

```diff
@@ -74,7 +74,7 @@ previous stable release.
 
 ### Foreman version compatibility notes
 
-This module targets Foreman 2.0+.
+This module targets Foreman 2.2+.
```

Comment:
We don't do this, see the support policy. Instead, we would write a new paragraph that on Foreman versions older than 2.2 you need to change the logging layout parameter to `pattern`.